src/Camera_TEST.cc

```diff
@@ -62,6 +62,18 @@ TEST(DOMCamera, Construction)
   cam.SetFarClip(200.2);
   EXPECT_DOUBLE_EQ(200.2, cam.FarClip());
 
+  cam.SetSegmentationType("semantic");
+  EXPECT_EQ("semantic", cam.SegmentationType());
+  EXPECT_TRUE(cam.HasSegmentationType());
+
+  cam.SetBoundingBoxType("3d");
```

Comment:
```suggestion
  EXPECT_EQ("2d", cam.BoundingBoxType())
  cam.SetBoundingBoxType("3d");
```
The bbox type is `2d` by default according to `sdf/1.9/camera.sdf`, so can you check that the default value is correct first before setting it to a different type?